### PR TITLE
Migrate the old '/Contributors' endpoint to v2

### PR DESCRIPTION
**Pull request by well-balanced**
# Description
Implements #286
- Call `v2/contributors` from the frontend
- Remove the old API `/contributors`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: web/src/apps/main/redux/actions/articles-page/index.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const fetchCurrentArticleContributors = (): ThunkResult<
   if (!currentArticle || Array.isArray(currentArticle.contributors)) return;
 
   const response = await Axios.get<GithubUser[]>(
-    apiURL + `/contributors?articleSlug=${currentArticle.slug}`,
+    apiURL + `/v2/contributors?articleSlug=${currentArticle.slug}`,
```
Comment (ZibanPirate) [inline web/src/apps/main/redux/actions/articles-page/index.ts:64]: as you can see in [here](https://github.com/dzcode-io/dzcode.io/blob/f54895bba3a432754d9174a4259e2c7999b4e61a/api/src/contributor/controller.ts#L27), the endpoint has changed a bit, now you should use it like this:

```suggestion
    apiURL + `/v2/contributors?path=articles/${currentArticle.slug}`,
```

File: web/src/apps/main/redux/actions/documentation-page/index.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ const fetchCurrentDocumentContributors = (): ThunkResult<
   if (!currentDocument || Array.isArray(currentDocument.contributors)) return;
 
   const response = await Axios.get<GithubUser[]>(
-    apiURL + `/contributors?documentSlug=${currentDocument.slug}`,
+    apiURL + `/v2/contributors?documentSlug=${currentDocument.slug}`,
```
Comment (ZibanPirate) [inline web/src/apps/main/redux/actions/documentation-page/index.ts:64]: Same here, but this time it's documentation:

```suggestion
    apiURL + `/v2/contributors?path=documentation/${currentDocument.slug}`,
```

**Review by ZibanPirate — CHANGES_REQUESTED** [review]
Great PR @well-balanced !

just apply the suggested changes, then we can merge 👌🏽

**Comment by well-balanced** [comment]
thank you for suggestion 🙏

**Review by ZibanPirate — APPROVED** [review]
Great work @well-balanced merging ...

**Pull request merged**